src/components/ha-sidebar.ts

```diff
@@ -292,106 +295,92 @@ class HaSidebar extends LitElement {
 
       <div class="divider"></div>
 
-      <div
-        class="notifications-container"
+      <mwc-list-item
+        class="notifications"
+        aria-role="option"
+        @click=${this._handleShowNotificationDrawer}
         @mouseenter=${this._itemMouseEnter}
         @mouseleave=${this._itemMouseLeave}
+        graphic="icon"
       >
-        <paper-icon-item
-          class="notifications"
-          aria-role="option"
-          @click=${this._handleShowNotificationDrawer}
-        >
-          <ha-svg-icon slot="item-icon" .path=${mdiBell}></ha-svg-icon>
-          ${!this.expanded && notificationCount > 0
-            ? html`
-                <span class="notification-badge" slot="item-icon">
-                  ${notificationCount}
-                </span>
-              `
-            : ""}
-          <span class="item-text">
-            ${hass.localize("ui.notification_drawer.title")}
-          </span>
-          ${this.expanded && notificationCount > 0
-            ? html`
-                <span class="notification-badge">${notificationCount}</span>
-              `
-            : ""}
-        </paper-icon-item>
-      </div>
-
-      <a
+        <ha-svg-icon slot="graphic" .path=${mdiBell}></ha-svg-icon>
+        ${!this.expanded && notificationCount > 0
+          ? html`
+              <span class="notification-badge" slot="graphic">
+                ${notificationCount}
+              </span>
+            `
+          : ""}
+        ${hass.localize("ui.notification_drawer.title")}
+        ${this.expanded && notificationCount > 0
+          ? html` <span class="notification-badge">${notificationCount}</span> `
+          : ""}
+      </mwc-list-item>
+      <mwc-list-item
         class=${classMap({
           profile: true,
           // Mimick behavior that paper-listbox provides
           "iron-selected": hass.panelUrl === "profile",
         })}
-        href="/profile"
-        data-panel="panel"
-        tabindex="-1"
+        .url=${`/profile`}
+        data-panel=${"profile"}
+        .activated=${this.hass.panelUrl === "profile"}
         aria-role="option"
         aria-label=${hass.localize("panel.profile")}
+        @click=${this._navigateToPanel}
         @mouseenter=${this._itemMouseEnter}
         @mouseleave=${this._itemMouseLeave}
+        graphic="avatar"
       >
-        <paper-icon-item>
-          <ha-user-badge
-            slot="item-icon"
-            .user=${hass.user}
-            .hass=${hass}
-          ></ha-user-badge>
-
-          <span class="item-text">
-            ${hass.user ? hass.user.name : ""}
-          </span>
-        </paper-icon-item>
-      </a>
-      <div disabled class="bottom-spacer"></div>
+        <ha-user-badge
+          .user=${hass.user}
+          .hass=${hass}
+          slot="graphic"
+        ></ha-user-badge>
+        ${hass.user ? hass.user.name : ""}
+      </mwc-list-item>
+      <mwc-list-item disabled class="bottom-spacer"></mwc-list-item>
```

Comment:
https://github.com/material-components/material-components-web-components/tree/master/packages/list#dividers

If you use a divider does it produce the same result?

Reply:
I saw that, but I assumed that dividers were just the little `<hr />` lines inbetween list items. I'll play with that.